climpred/metrics.py

```diff
@@ -90,6 +89,16 @@ def _preprocess_dims(dim):
     return dim
 
 
+def _sanitize_kwargs(kwargs, delete=['comparison', 'alignment']):
```

Comment:
Generally want to avoid having a list as a default, as it causes unintended behavior (https://docs.python-guide.org/writing/gotchas/).

I would do

```python
 def _sanitize_kwargs(kwargs, delete=None):
    if delete is None:
        delete = ['comparison', 'alignment']
```